Build failure: clamav

### Steps To Reproduce

Steps to reproduce the behavior:
1. build *clamav*

### Build log

```
error: builder for '/nix/store/z8mbbp1n6axb1rw15i20niiskh0jiq6k-clamav-1.3.1.drv' failed with exit code 2;
       last 25 log lines:
       >
       > [DEBUG]: stderr:
       >
       > =========================== short test summary info ============================
       > FAILED clamscan/assorted_test.py::TC::test_pe_cert_trust - AssertionError: as...
       > ======================== 1 failed, 52 passed in 14.48s =========================
       >
       > 3/5 Test #4: clamd ............................   Passed   19.33 sec
       > 4/5 Test #1: libclamav ........................   Passed   23.96 sec
       > 5/5 Test #2: libclamav_rust ...................   Passed   28.29 sec
       >
       > 80% tests passed, 1 tests failed out of 5
       >
       > Total Test time (real) =  28.31 sec
       >
       > The following tests FAILED:
       >        3 - clamscan (Failed)
       > Errors while running CTest
       > make[3]: *** [CMakeFiles/check.dir/build.make:70: CMakeFiles/check] Error 8
       > make[3]: Leaving directory '/build/clamav-1.3.1/build'
       > make[2]: *** [CMakeFiles/Makefile2:1348: CMakeFiles/check.dir/all] Error 2
       > make[2]: Leaving directory '/build/clamav-1.3.1/build'
       > make[1]: *** [CMakeFiles/Makefile2:1355: CMakeFiles/check.dir/rule] Error 2
       > make[1]: Leaving directory '/build/clamav-1.3.1/build'
       > make: *** [Makefile:563: check] Error 2
       For full logs, run 'nix log /nix/store/z8mbbp1n6axb1rw15i20niiskh0jiq6k-clamav-1.3.1.drv'.
```

### Additional context

Happened during last system update.

### Notify maintainers
@robberer @qknight @globin

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.44, NixOS, 24.11 (Vicuna), 24.11.20240811.a58bc8a`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.91.0`
 - nixpkgs: `/nix/store/h60m1fwahjd2mv6gsg77ji3vb4gpj4dk-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/tools/security/clamav/default.nix
+++ b/pkgs/tools/security/clamav/default.nix
@@ -6,11 +6,11 @@
 
 stdenv.mkDerivation rec {
   pname = "clamav";
-  version = "1.3.1";
+  version = "1.4.0";
 
   src = fetchurl {
     url = "https://www.clamav.net/downloads/production/${pname}-${version}.tar.gz";
-    hash = "sha256-EqMDW/JvVfceMQalGl+o17dEVy35imOSCpz/h2p9zOQ=";
+    hash = "sha256-1nqymeXKBdrT2imaXqc9YCCTcqW+zX8TuaM8KQM4pOY=";
   };
 
   patches = [

